charts/simple-app/values.yaml

```diff
@@ -110,17 +110,11 @@ startupProbe:
 # livenessProbe will be applied to the proxySidecar container instead if that
 # is enabled.
 livenessProbe:
-  httpGet:
```

Comment:
Convert type of `livenessProbe` from `null` to empty object `{}`. I do not remember exactly why/how this works, but I do remember that we do not want to do multi-typing.

Let me also ask #cloudeng about this one too as I am not totally sure what I am remembering here is applicable to this specific scenario.

Reply:
I copied `startupProbe` above.

Reply:
⛔️ Blocking comment.

Reply:
going to leave as is and all probes can be updated in a followup PR following best typing practices.